components/shared/gallery/BreadcrumbsFilter.vue

```diff
@@ -1,32 +1,62 @@
 <template>
   <b-field grouped group-multiline class="filters-tag">
-    <template v-for="(value, key) in breads">
-      <div v-if="value === 'true'" :key="key" class="control">
-        <!-- NeoTag -->
-        <NeoTag @close="closeTag(String(key))">
-          {{ queryMapTranslation[String(key)] }}
-        </NeoTag>
-      </div>
+    <template v-if="true">
+      <template v-for="(value, key) in breads">
+        <div v-if="value === 'true'" :key="key" class="control">
```

Comment:
have a look again please
it's not exactly as you suggested, but pretty close and neater then before
thank you

Reply:
why `v-else` doesn't work? is there a new specific case?

Reply:
yes
when there is no search.
in that case  ```v-if="key === 'search' && value != undefined"``` falls through, because the the value is undefined
and using just ```v-else``` renders for the empty search